readme.md

```diff
@@ -1,7 +1,17 @@
 # Readme.md
 
-This file has notes on building the API reference
-for the Roslyn APIs.
+This repository contains the API documentation for the
+[.NET Compiler Platform SDK](https://github.com/dotnet/roslyn-sdk). The .NET
```

Comment:
I think it would be better to link to dotnet/roslyn instead, since that's where the source for the API is.

Reply:
I'd like to get @jinujoseph, @kuhlenh or @kathleendollard to weigh in on that.

Which would be the best link?

Reply:
After checking internally, the best answer is "both".

Reply:
I think this would be a great place to clarify what can be confusing differences. Maybe:

This repository contains documentation for the [Roslyn platform](https://github.com/dotnet/roslyn) and the [.NET Compiler Platform SDK](https://github.com/dotnet/roslyn-sdk). The Roslyn platform contains the compiler and is part of every .NET installation. The Compiler Platform SDK is available via NuGet and provides tools,  templates and samples to access to the Roslyn platform.

Reply:
> and is part of every .NET installation

every .NET Core installation.  .NET Framework does not require the compilers

> The Compiler Platform SDK is available via NuGet and provides tools, templates and samples to access to the Roslyn platform.

it doesn't provide any nuget packages, those come from the dotnet/roslyn repo.  Today the roslyn sdk only provides visual studio templates and tools

Reply:
Yes, you are correct on both points.  Thank you @jmarolf. 

Maybe this:

The Compiler Platform SDK provides tools and templates for programmers working with the  Roslyn platform.